pkg/spoke/controllers/lease/lease_controller.go

```diff
@@ -0,0 +1,73 @@
+package lease
```

Comment:
I do not think we need this for the current design?

Reply:
Removed.